google/cloud/internal/version_info.h

```diff
@@ -18,7 +18,7 @@
 // NOLINTNEXTLINE(modernize-macro-to-enum)
 #define GOOGLE_CLOUD_CPP_VERSION_MAJOR 2
 // NOLINTNEXTLINE(modernize-macro-to-enum)
-#define GOOGLE_CLOUD_CPP_VERSION_MINOR 23
+#define GOOGLE_CLOUD_CPP_VERSION_MINOR 24
 // NOLINTNEXTLINE(modernize-macro-to-enum)
 #define GOOGLE_CLOUD_CPP_VERSION_PATCH 0
 #define GOOGLE_CLOUD_CPP_VERSION_PRE_RELEASE ""
```

Comment:
"rc"

and regenerate the ABI dumps

Reply:
It was generated with the incremented version #. The merge commit overwrote those changes.

Sorry it's confusing.

Reply:
https://github.com/googleapis/google-cloud-cpp/pull/13995/commits/991480b4806dd007fe259984bd39e0e14a0e3a84#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL26

Reply:
Ok maybe I did not generate this correctly. Gonna start on a fresh PR